src/libs/actions/IOU.js

```diff
@@ -509,6 +510,10 @@ function getMoneyRequestInformation(
  * @param {String} merchant
  */
 function createDistanceRequest(report, participant, comment, created, transactionID, amount, currency, merchant) {
+    const optimisticReceipt = {
+        source: ReceiptGeneric,
```

Comment:
Can we use the[ map pending image](https://github.com/Expensify/App/blob/f0243c4785868c036b9bc3d5aa843aa03975325b/src/components/DistanceRequest.js#L254) instead?

Reply:
> From my testing, image doesn't show at all when offline, even though local image.

Yeah it doesn't look like these images are pre-cached so it doesn't load while offline on web:

![Screenshot 2023-09-04 at 3 57 49 PM](https://github.com/Expensify/App/assets/6833644/320e225a-e493-4b0d-9a77-5e49357eb02f)

Let me try the icon